src/components/design-library-list/design-preview.js

```
@@ -0,0 +1,43 @@
+	] ), {
+		'preview-pages': selectedTab === 'pages',
+	} )
+
+	return createPortal( <>
+		<body
+			ref={ ref }
+			className={ shadowBodyClasses }
+			onWheel={ onScroll }
+		>
+			<div
+				dangerouslySetInnerHTML={ { __html: blocks } }
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)